types/libpq/tslint.json

```diff
@@ -2,6 +2,7 @@
     "extends": "dtslint/dt.json",
     "rules": {
         "ban-types": false,
-        "no-void-expression": false
+        "no-void-expression": false,
+        "no-outside-dependencies": false
```

Comment:
Why is this needed?

Reply:
https://github.com/microsoft/dtslint/issues/315